nanodbc/nanodbc.cpp

```diff
@@ -2924,14 +2924,18 @@ inline void result::result_impl::get_ref_impl(short column, T& result) const
                     buffer,          // TargetValuePtr
                     buffer_size,     // BufferLength
                     &ValueLenOrInd); // StrLen_or_IndPtr
-                if (ValueLenOrInd == SQL_NO_TOTAL)
-                    out.append(buffer, col.ctype_ == SQL_C_BINARY ? buffer_size : buffer_size - 1);
-                else if (ValueLenOrInd > 0)
-                    out.append(
-                        buffer,
-                        std::min<std::size_t>(
-                            ValueLenOrInd,
-                            col.ctype_ == SQL_C_BINARY ? buffer_size : buffer_size - 1));
+                if (ValueLenOrInd == SQL_NO_TOTAL || ValueLenOrInd > 0) {
```

Comment:
Your patch changes coding style/formatting, See the first Travis CI build job. You may need to run `clang-format` - `.clang-format` is provided in the tree.

Reply:
Sorry for the formatting change. I use VSCode and disables my local `formatOnSave`. Thanks for noticing `.clang-format` file. I'll configure my extension to respect this file.